openfisca_us/variables/gov/irs/credits/cdcc/head_earned.py

```diff
@@ -0,0 +1,15 @@
+from openfisca_us.model_api import *
+
+
+class head_earned(Variable):
```

Comment:
Could also name `head_nonnegative_earned` given the operation

Reply:
Can we use `positive` instead of `nonnegative`?

Reply:
It can be zero which isn't positive